extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -537,7 +537,7 @@ export class AmpSelector extends AMP.BaseElement {
   selectionKeyDownHandler_(event) {
     const {key} = event;
     if (key == Keys.SPACE || key == Keys.ENTER) {
-      if (this.elements_.includes(event.target)) {
+      if (this.elements_.includes(dev().assertElement(event.target))) {
```

Comment:
I am not convinced `event.target` can't be null here in legit cases.  `includes` don't accept `null`-able`? seems